src/js/constants.js

```diff
@@ -114,6 +114,19 @@ export const availableLayers = [
   "NICFI",
 ];
 
+export const layerColors = {
+  cMines: "var(--confirmed-mines)",
+  nMines: "var(--new-mines)",
+  pMines: "var(--current-pred)",
+  municipalBounds: "var(--municipal-bounds)",
```

Comment:
I think we can just put the colors hex here, since this is already a constnat (defined in js) without having antoher level of indirection.